### PR TITLE
Update appliance about page content

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Update appliance about page content

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/about
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the content matches [the copy doc](https://docs.google.com/document/d/1UM4K9n9SFNiWdmKIBnPGCsjLQrPSOnqZHMNIMwQ2CZI/edit#)

### CODE REVIEW

File: templates/appliance/about.html [hunk 1]
```diff
@@ -2,117 +2,151 @@
...
   </div>
 </section>
 
 <section class="p-strip">
   <div class="row">
     <div class="col-8">
-      <h2>Automatic system updates for ten years</h2>
-      <p>The underlying operating system of all Ubuntu Appliance images is Ubuntu Core, the highly secure, fully containerised Linux platform that uses transactional update mechanisms to ensure it can always roll back to a known-good state. The platform elements of Ubuntu Core are maintained for ten years, so Ubuntu Appliances can meet enterprise security and longevity requirements.</p>
+      <h2>Security and simplicity, by design</h2>
+      <p>The Ubuntu Appliance goal is automatic widespread security without human supervision, in every building and on every street. Security is a many-layered endeavour - Ubuntu Appliances respond with defence in depth. Every aspect of an Ubuntu Appliance is built with security in mind:</p>
```
Comment (steverydz) [inline templates/appliance/about.html:70]: The `-` should be `&emdash;`

File: templates/appliance/about.html [hunk 2]
```diff
@@ -2,117 +2,151 @@
 
 {% block title %}About Ubuntu Appliances{% endblock %}
```
Comment (steverydz) [inline templates/appliance/about.html:3]: Title should be "What is an Ubuntu Appliance?"

File: templates/appliance/about.html (hunk 1 above)
Comment (anthonydillon) [reply templates/appliance/about.html:70]: done

File: templates/appliance/about.html (hunk 2 above)
Comment (anthonydillon) [reply templates/appliance/about.html:3]: done

**Pull request merged**